components/admin_console/system_user_detail/index.js

```diff
@@ -14,10 +15,12 @@ import {getNavigationBlocked, showNavigationPrompt} from 'selectors/views/admin'
 import SystemUserDetail from './system_user_detail.jsx';
 
 function mapStateToProps(state, ownProps) {
+    const config = getConfig(state);
     const userId = ownProps.match.params.user_id;
     const user = getUser(state, userId);
     return {
         user,
+        mfaEnabled: config.EnableMultifactorAuthentication === 'true',
```

Comment:
Yes, it will always be a lowercase string. The config util helps with that. Also, if you look through the codebase this pattern is followed everywhere with the config. Thanks for checking.